images/provisioner/setup.sh

```diff
@@ -30,5 +30,5 @@ if [ ! -f "$FILE" ]; then
   sudo apt update && sudo apt install -y nvidia-docker2
   sudo systemctl restart docker
 
-  sudo echo 'OK' >"$FILE"
+  echo OK | sudo tee "$FILE"
```

Comment:
was not this addressed at #205 ?

Reply:
Yes, please notice that this is a **draft** pull request and may include temporary upstream patches. 🙃